alembic/versions/f5716ac94693_dropping_mpi_schema.py

```diff
@@ -0,0 +1,55 @@
+"""dropping mpi schema
+
+Revision ID: f5716ac94693
+Revises: 591a56cee781
+Create Date: 2024-09-25 15:47:22.497271
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import sqlite
+
+# revision identifiers, used by Alembic.
+revision: str = 'f5716ac94693'
+down_revision: Union[str, None] = '591a56cee781'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('mpi_patient')
```

Comment:
I wanted to make some adjustments to the primary keys in the MPI tables.  Because sqlite doesn't support altering column types, the easiest path forward is to just drop the tables, then recreate them.